kie-server-parent/kie-server-common/src/test/java/org/kie/server/common/KeyStoreHelperUtilTest.java

```diff
@@ -26,6 +26,16 @@
 
 public class KeyStoreHelperUtilTest {
 
+    @Test
+    public void testMultipleLoadsPassword() {
+        final String defaultPassword = "default";
+        final String otherDefaultPassword = "otherDefault";
+        String password = loadControllerPassword(defaultPassword);
+        assertEquals(defaultPassword, password);
+        password = loadControllerPassword(otherDefaultPassword);
+        assertEquals(otherDefaultPassword, password);
+    }
```

Comment:
Not sure this test is even needed. Is there any special case why we need to test it twice in a row?

Reply:
I did the test to check on local the message was not don twice. I will get rid of it.